scrapers/FuckedHard18.yml

```
@@ -0,0 +1,34 @@
+name: "FuckedHard18"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - fuckedhard18.com/membersarea/gallery.php
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: &name
+        selector: //a[@class="model_category_link"][contains(@href,'sets.php')]/text()
+      Details:
+      Date:
+        selector: //td[@class="date"]/text()
+        postProcess:
+          - replace:
+            - regex: Added 
+              with: ""
+          - parseDate: 01/02/2006 
+      Image: 
+        selector: //a[@class="jqModal"]/img[@class="thumbs"]/@src[1]
```

This returns all the `idividual clip` images. ( of which stash needs only 1 so gets the  first)
If we only want the first (i also cant find an alternative for a better quality image), to be more exact
```suggestion
        selector: (//a[@class="jqModal"]/img[@class="thumbs"])[1]/@src
```
should be used instead

---

Thanks! Yes, unfortunately the data on the website is quite poor also for title and tags. In most cases the thumb generated by stash is better.